petab/sampling.py

```diff
@@ -0,0 +1,103 @@
+"""Functions related to parameter sampling"""
+
+import numpy as np
+import pandas as pd
+
+from typing import Tuple
+
+from .parameters import get_priors_from_df
+
+
+def sample_from_prior(prior: Tuple[str, list, str, list],
```

Comment:
nice convenience function. I'd suggest allowing to pass an optional seed if reproducibility is desired.

Reply:
Has already been there, just moved to a more prominent spot. Can add.

Reply:
let's do that separately then

Reply:
Too late ;)